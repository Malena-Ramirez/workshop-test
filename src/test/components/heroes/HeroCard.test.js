import React from 'react';
import Adapter from 'enzyme-adapter-react-16';
import { shallow, configure } from 'enzyme';
import { HeroCard } from '../../../components/heroes/HeroCard';

configure({ adapter: new Adapter() });

describe('Pruebas en <HeroCard/>', () => {
  test('Validar la informaciÃ³n de las tarjetas', () => {
    const id = 'dc-batman';
    const superhero = 'Batman';
    const alter_ego = 'Bruce Wayne';
    const first_appearance = 'Detective Comics #27';
    const characters = 'Bruce Wayne';

    const wrapper = shallow(<HeroCard
      id={id}
      superhero={superhero}
      alter_ego={alter_ego}
      first_appearance={first_appearance}
      characters={characters}
    />)

    const textParrafo = wrapper.find('h5').text().trim();

    expect(wrapper).toMatchSnapshot();
    expect(textParrafo).toBe('Batman');
  })

})
