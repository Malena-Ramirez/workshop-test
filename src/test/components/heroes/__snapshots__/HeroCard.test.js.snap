// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <HeroCard/> Validar la informaci√≥n de las tarjetas 1`] = `ShallowWrapper {}`;
